channel-bot.py

```diff
@@ -36,11 +37,32 @@ def convert_channels_to_text(channels):
     return "".join(text_list).encode("utf-8")
 
 
+def read_config(config):
+    error_message = "Please create 'slack' section and contain options."
+    try:
+        config.read("config.ini")
+    except MissingSectionHeaderError:
+        print error_message
+        exit()
```

Comment:
`exit()` is intended to use in interactive shell.
So you should use `sys.exit()` instead of `exit()` in code.